### DISCUSSION TITLE
Will it be available on Windows or Linux in the future?

### DISCUSSION THREAD

**Discussion by java-github-dotcom** [ggml-org/llama.cpp#10248] [General]
Is there any plan to make it available on Windows or Linux platforms in the future? If so, what is the expected timeline for release and will there be any specific features or differences compared to the current version?

**Comment by wooooyeahhhh** [comment]
What? almost all the features of llama.cpp is available on linux and mac and windows, the only differences depend on the backend used

**Reply by java-github-dotcom** [reply]
oh really? sorry just had a quick glance and noticed only the part associated to macs. thanks for clarifying
